transports/msmq/transportconfig_usenontransactionalqueues_msmqtransport_[1,).partial.md

```diff
@@ -0,0 +1,13 @@
+
+### UseNonTransactionalQueues
+
+This setting should be used with caution. As the queues are not transactional, any message that has an exception during processing will not be rolled back to the queue. Therefore this setting must only be used where loss of messages is an acceptable scenario. 
```

Comment:
Perhaps a note that this is not the same as transactionmode.none and that all endpoints needs to agree on this setting to be able to comminicate?